File: golang/codewars.go
package main

import (
	"strings"
)

func EvenOrOdd(number int) string {
	if number%2 == 0 {
		return "Even"
	} else {
		return "Odd"
	}
}

func CountSheeps(numbers []bool) int {
	var count = 0
	for _, value := range numbers {
		if value {
			count = count + 1
		}
	}
	return count
}

func monkeyCount(n int) []int {
	var monkey []int
	monkeySlice := monkey[:]
	for i := 1; i <= n; i++ {
		monkeySlice = append(monkeySlice, i)
	}
	return monkeySlice
}

func PaperWork(n1 int, m int) int {
	if n1 > 0 && m > 0 {
		return (n1 * m)
	} else {
		return 0
	}
}

func Hero(bullets, dragons int) bool {
	if bullets >= (dragons * 2) {
		return true
	} else {
		return false
	}
}

func correctPolishLetters(TextPolish string) string {
	replacer := strings.NewReplacer("ą", "a", "ć", "c", "ę", "e", "ł", "l", "ń", "n", "ó", "o", "ś", "s", "ź", "z", "ż", "z")
	TextRus := replacer.Replace(TextPolish)
	return TextRus
}

func findAll(n int, massiv []int) []int {
	var out []int
	for i, num := range massiv {
		if num == n {
			out = append(out, i)
		}
	}
	return out
}

func sumOfMinimums(matrix [][]int) int {
	var summa int
	for _, scores := range matrix {
		for _, num := range scores {
			if num < scores[0] {
				scores[0] = num
			}
		}
		summa += scores[0]
	}
	return summa
}

File: python/codewars.py
# task 1
def even_or_odd(number):
    if number % 2 == 0:
        return "Even"
    else:
        return "Odd"


# task 2
def count_sheeps(sheep):
    return sheep.count(True)


# task 3
def monkey_count(n):
    result = []
    for num in range(1, n + 1):
        result.append(num)
    return result


# task 4
def paperwork(n, m):
    return n * m if n > 0 and m > 0 else 0


# task 5
def hero(b, d):
    bulletsneeded = d * 2
    if b >= bulletsneeded:
        return True
    else:
        return False


# task 1
def correct_polish_letters(st):
    pol = {"ą": "a", "ć": "c", "ę": "e", "ł": "l", "ń": "n", "ó": "o", "ś": "s", "ź": "z", "ż": "z"}
    return "".join([pol[i] if i in pol else i for i in st])


# task 2
def find_all(array, n):
    arr = []
    for i in range(len(array)):
        if array[i] == n:
            arr.append(i)
    return arr


# task 3
def sum_of_minimums(numbers):
    return sum(map(min, numbers))
